server/api.js

```diff
@@ -101,21 +101,30 @@ router.get("/quizes/search",(req, res)=>{
 	});
 });
 
-router.get("/results/search",(req, res)=>{
+router.get("/results/:id",(req, res)=>{
 	const client = getClient();
 	client.connect(()=>{
 		const db = client.db("quiz");
 		const collection = db.collection("results");
-		const quiz = { name:req.query.name };
-		collection.find(quiz).toArray((err, results)=>{
+
+		const quizId = req.params.id;
+		if (!mongodb.ObjectID.isValid(quizId)) {
+			return res.status(400).json("the 'Quiz Id' is not valid");
+		}
+		// const id = mongodb.ObjectID(quizId);
```

Comment:
This can probably go?

Reply:
so it's not the new changes because in new changes I removed the comment.